src/components/Forms/CreateMemberForm/CreateMemberForm.jsx

```diff
@@ -2,23 +2,27 @@ import React from 'react';
 import { Form } from 'react-bootstrap';
 import propTypes from 'prop-types';
 import { initialStateCreatMember } from '../../../shared/initialStates';
-import { BUTTONS_TYPES, BUTTONS_NAMES, USER_FIELDS_KEYS } from '../../../shared/constants';
+import { BUTTONS_TYPES, BUTTONS_NAMES, USER_FIELDS_KEYS, FORM_MEMBER_ERRORS } from '../../../shared/constants';
 import { Button } from '../../Buttons/Button/Button';
 import { FormField } from '../FormField/FormField';
 import style from './CreateMemberForm.module.css';
 import { validateFormField } from '../../../shared/helpers/validateFormField/validateFormField';
+import { typesValidation } from '../../../shared/helpers/typesValidation/typesValidation';
 
 export class CreateMemberForm extends React.PureComponent {
   constructor(props) {
     super(props);
-    this.state = initialStateCreatMember;
+    this.state = { ...initialStateCreatMember, formErrors: FORM_MEMBER_ERRORS };
   }
 
   async componentDidMount() {
     const { userData, isEditMode } = this.props;
 
     if (isEditMode) {
-      this.setState({ ...userData });
+      this.setState({
+        ...userData,
+        formErrors: FORM_MEMBER_ERRORS.map((item) => ({ ...item, error: '' })),
```

Comment:
change this logic, you don't need to reset errors on mount

Reply:
fixed.
```suggestion
  async componentDidMount() {
    const { userData, isEditMode } = this.props;

    if (isEditMode) {
      this.setState({
        ...userData,
      });
    }
  }
```

```
  handleSubmit = async (e) => {
    e.preventDefault();
    const { isEditMode, createUserHandler, editUserDataHandler } = this.props;
    const { formErrors, ...data } = this.state;
    const isError = formErrors
      .map((item) => {
        const { name, error } = validateFormField(item.name, data[item.name], data.password);
        this.setState((prevState) => ({
          ...prevState,
          formErrors: prevState.formErrors.map((field) => (field.name === name ? { ...field, error } : field)),
        }));

        return error;
      })
      .filter((error) => error);
    if (isEditMode && !isError.length) {
      await editUserDataHandler(userTypesValidation(data));
    } else if (!isError.length) {
      await createUserHandler(userTypesValidation(data));
    }
  };
```